third_party/istio-latest

```diff
@@ -0,0 +1 @@
+istio-1.4-latest
```

Comment:
<!--whitespace-bot-->
Add trailing newline:
```suggestion
istio-1.4-latest

```

Reply:
Can't do it for symlink I think